WordPress/src/main/java/org/wordpress/android/ui/posts/PostPreviewActivity.java

```diff
@@ -151,6 +156,17 @@ private void refreshPreview() {
     }
 
     @Override
+    protected void onRestoreInstanceState(Bundle savedState) {
```

Comment:
The state is already restored in `onCreate()` so having `onRestoreInstanceState()` is redundant.